packages/kbn-securitysolution-utils/src/path_validations/index.ts

```diff
@@ -101,9 +101,17 @@ export const validateFilePathInput = ({
   }
 };
 
-export const validateWildcardInput = (value?: string): string | undefined => {
-  if (/[*?]/.test(value ?? '')) {
-    return WILDCARD_WARNING;
+export const doesAnyValueContainWildcard = (value: string | string[]): string | undefined => {
```

Comment:
sorry for the misunderstanding - this function had a proper name called `validateWildcardInput`, similarly to this file's other functions which _validate_ something and return either `undefined` or a warning message (and not boolean as the 'does...' prefix would suggest)

in my previous comment i meant the `test` variable in line 107, but it's not a biggie, it works with `test`, too, so just a nitpicking

however, could you restore the original name of the function to `validateWildcardInput`, just to keep the similar naming of the similar functions in the file?